service-manual/the-team/learning-and-development/foundation-day.md

```diff
@@ -87,5 +87,5 @@ As preparation before the day, we ask that participants:
 
 To request a place on the foundation day, please fill in this [Civil Service Learning request form](https://civilservicelearning.civilservice.gov.uk/learning-opportunities/digital-foundation-day)
 
-Any questions? Please contact the GDS Skills Team: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:service-manager-programme@digital.cabinet-office.gov.uk)
+Any questions? Please contact: [service-manager-programme@digital.cabinet-office.gov.uk](mailto:digital.academy@dwp.gsi.gov.uk)
```

Comment:
Thanks for this! A quick thing though: you'll need to change the text between the `[]` that currently advertises a CO address, as the actual email will go to a DWP address.